src/tests/ftest/io/nvme_io.yaml

```diff
@@ -12,6 +12,8 @@ hosts:
 timeout: 28800
 server_config:
  name: daos_server
+ bdev_class: nvme
```

Comment:
So this test now requires NVMe hardware, correct?  If that is the case, you probably want a tag on the test that specifies that it should only be run with NVMe installed hardware.  @skirvan44: Do you have a tag for that yet?

Reply:
Changing tag to "hw"